iron-input.html

```diff
@@ -68,18 +68,20 @@
       },
 
       /**
-       * Set to true to prevent the user from entering invalid input. The new input characters are
-       * matched with `allowedPattern` if it is set, otherwise it will use the `type` attribute (only
-       * supported for `type=number`).
+       * Set to true to prevent the user from entering invalid input. If `allowedPattern` is set, 
+       * The new input characters are matched with that. If more than one character is typed or pasted in,
```

Comment:
I like your suggestion to clarify typed in characters and your wording for pasting text, but I'm afraid I don't understand something. When I paste text into an `iron-input` that has non-valid characters, the entire thing is *rejected*, rather than accepted. Unless I'm missing something?

(You can quickly verify this on the [paper-input demo](https://elements.polymer-project.org/elements/paper-input?view=demo:demo/index.html&active=paper-input) by pasting '123' into the input labeled "this input will only let you type letters")